bin/common.sh

```diff
@@ -49,7 +51,7 @@ function delete_namespaces {
 
 function apply_manifests() {
   oc apply -f tests/custom-pidslimit.yaml || return $?
-  oc label machineconfigpools.machineconfiguration.openshift.io worker custom-crio=custom-pidslimit
+  oc label machineconfigpools.machineconfiguration.openshift.io worker custom-crio=custom-pidslimit --overwrite
```

Comment:
Get rid of stupid warning (kind of irrelevant to the PR)

Reply:
I remember doing this patch locally but then I've never sent the PR, thanks!